community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables_controller_instance.tf

```diff
@@ -91,10 +101,20 @@ variable "can_ip_forward" {
   default     = false
 }
 
-variable "disable_controller_public_ips" {
-  description = "If set to false. The controller will have a random public IP assigned to it. Ignored if access_config is set."
+variable "enable_controller_public_ips" {
+  description = "If set to true. The controller will have a random public IP assigned to it. Ignored if access_config is set."
   type        = bool
-  default     = true
+  default     = false
+}
+
+variable "disable_controller_public_ips" { # tflint-ignore: terraform_unused_declarations
+  description = "DEPRECATED: Use `enable_controller_public_ips` instead."
+  type        = bool
+  default     = null
```

Comment:
what would happen if we explicitly mention disable_controller_public_ips: false in the blueprint since condition only checks if it is null or not. Would that pass ?

Reply:
It will fail if it mentioned , regardless of the supplied value:
* `false | true` - condition `== null` will fire;
* `null` - type checker will complain "can't use null here, it's bool"